### ISSUE TITLE
Hypoplasia of corpus cavernosum

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9562]
**Preferred term label:**
Corpus cavernosum hypoplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
615542 
PMID: 21220346   1/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
